src/Razor/src/Microsoft.AspNetCore.Razor.Language/Intermediate/ComponentIntermediateNode.cs

```diff
@@ -18,6 +15,8 @@ public sealed class ComponentIntermediateNode : IntermediateNode
 
         public IEnumerable<SetKeyIntermediateNode> SetKeys => Children.OfType<SetKeyIntermediateNode>();
 
+        public IEnumerable<SplatIntermediateNode> Splats => Children.OfType<SplatIntermediateNode>();
```

Comment:
Hmm, do we use this anywhere?

Reply:
No, but it would be inconsistent to leave it out, so I did it.